README.md

```diff
@@ -321,15 +344,43 @@ Take your time to plan out your code, test each method, and make many commits. A
 
 ### Phase 4: Project Review
 
-When you are ready, schedule a project review with one of your instructors. The format of the review will roughly look like this:
+Communicate with your instructor to schedule a project review. The format of the review will roughly look like this:
 
-* 3 minutes - Requirements Review
+* 5 minutes - Requirements Review
 * 5 minutes - App Overview
-* 6 minutes - Deep Dive
-* 12 minutes - Pair Programming
-* 4 minutes - Feedback
+* 15 minutes - Deep Dive
+* 15 minutes - Pair Programming
+* 5 minutes - Wrap up
+
+In the interview, the instructor will check that your project meets the deliverables, have you explain your code, and check your understanding of the concepts through pair programming. 
+
+They'll use the following rubric to evaluate the project and interview.
+
+#### Rubric
+
+##### Designing a CLI Application
```

Comment:
This section seems to couple 2 things.
- Does it work from user pov and meet guidelines
- Implementation details of app

For example, I have definitely seen even really solidly implemented projects where
> and the CLI interface is separated from the models in the domain
is followed very inconsistently in different parts of the app

Reply:
I'd propose one of
* splitting this up into 2 sections 
* maybe lumping some of the "how is it implemented" into the OO Ruby Syntax section and renaming that since a bunch of it is just about well architected code anyway
* maybe just moving "separation of CLI and models" to the score of 4 or 5, because I'd guess many folks don't 100% hit that criteria

Reply:
Good feedback, I think either could work! Do you like one or the other better?